Makefile

```diff
@@ -1,7 +1,12 @@
-# Variables are documented in hack/build.sh.
+# Variables are documented in common/build.sh.
 BASE_IMAGE_NAME = postgresql
 VERSIONS = 9.2 9.4 9.5
 OPENSHIFT_NAMESPACES = 9.2
 
 # Include common Makefile code.
-include hack/common.mk
+-include common/common.mk
+git_submodule_warning:
```

Comment:
Because until you init the submodule, you don't have common.mk.  That target is invoked only if the 'make' is called and the git submodule is not initialized.